openfecwebapp/templates/partials/candidate/financial-summary.html

```diff
@@ -55,8 +55,8 @@ <h4 class="callout__title">
         </div>
         <div class="callout__action">
           <ul class="callout__sublinks">
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='receipts') }}">Money received</a></li>
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='disbursements') }}">Money spent</a></li>
+            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='receipts') }}">Receipts</a></li>
+            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='disbursements') }}">Disbursements</a></li>
```

Comment:
I get why you're changing this for consistency, but should we want to come up with some rules for when we want to say "money received" and "money spent", which seem easier to understand?